torchdynamo/eval_frame.py

```diff
@@ -157,6 +159,9 @@ def __call__(self, *args, **kwargs):
 
         @functools.wraps(fn)
         def _fn(*args, **kwargs):
+            if self.first_ctx:
+                log.info("torchdynamo begin tracing")
+
```

Comment:
Also since TorchDynamo + TorchInductor is a deep stack, user will benefit from a blueprint of the steps (e.g. what Steps we will go through and what each step does) in a short summary. Maybe we can put it as part of README.md file?